access_point/server/server.py

```diff
@@ -16,34 +16,41 @@ def index():
 def install():
     return 'starting the installation script'
 
-@app.route('/config/<stt>/<tts>/<hotword>/<wake>')
-def config(stt, tts, hotword, wake):
-    subprocess.call(['sudo', 'bash' , access_point_folder + '/config.sh', stt, tts, hotword, wake])  #nosec #pylint-disable type: ignore
-    display_message = {"Configuration":"Successful"}
+@app.route('/config', methods=['GET'])
+def config():
+    stt = request.args.get('stt')
+    tts = request.args.get('tts')
+    hotword = request.args.get('hotword')
+    wake = request.args.get('wake')
+    subprocess.Popen(['sudo','bash',access_point_folder + '/config.sh ',stt, tts, hotword, wake])  #nosec #pylint-disable type: ignore
```

Comment:
Python coding style!

Reply:
@hongquan , I have VSCode with pep 8 and pylint installed. I am not sure why it is not showing any errors. But I'll update it

Reply:
@stealthanthrax 
> I am not sure why it is not showing any errors

Maybe because of the directive `#nosec #pylint-disable type: ignore` in the line?